spec/indexers/plum_jsonld_converter_spec.rb

```diff
@@ -3,49 +3,49 @@
-  end
+  let(:jsonld) { file_fixture("plum_records/b9e8325e-baf2-45e4-b32c-5e5b3755c8ef.jsonld").read }
   before do
-    allow(converter).to receive(:open).with("https://hydra-dev.princeton.edu/concern/ephemera_folders/sx920fz824/manifest").and_return(
-      '{"@context":"http://iiif.io/api/presentation/2/context.json","@id":"https://hydra-dev.princeton.edu/concern/ephemera_folders/sx920fz824/manifest","@type":"sc:Manifest","label":["Test Folder 1"],"viewingHint":"individuals","viewingDirection":"left-to-right","sequences":[{"@id":"https://hydra-dev.princeton.edu/concern/ephemera_folders/sx920fz824/manifest/sequence/normal","@type":"sc:Sequence","canvases":[{"@id":"https://hydra-dev.princeton.edu/concern/ephemera_folders/sx920fz824/manifest/canvas/st722hc03g","@type":"sc:Canvas","label":"color-landscape.tif","images":[{"@id":"https://hydra-dev.princeton.edu/concern/ephemera_folders/sx920fz824/manifest/annotation/st722hc03g-image","@type":"oa:Annotation","motivation":"sc:painting","resource":{"@id":"https://libimages1.princeton.edu/loris/plum/st%2F72%2F2h%2Fc0%2F3g-intermediate_file.jp2/full/!600,600/0/default.jpg","@type":"dctypes:Image","format":"image/jpeg","width":287,"height":200,"service":{"@context":"http://iiif.io/api/image/2/context.json","@id":"https://libimages1.princeton.edu/loris/plum/st%2F72%2F2h%2Fc0%2F3g-intermediate_file.jp2","profile":"http://iiif.io/api/image/2/level2.json"}},"on":"https://hydra-dev.princeton.edu/concern/ephemera_folders/sx920fz824/manifest/canvas/st722hc03g"}],"width":287,"height":200}],"viewingHint":"individuals"}],"metadata":[{"label":"Creator","value":["Test Creator"]},{"label":"Contributors","value":["Test Contributor"]},{"label":"Published/Created","value":["Test Publisher"]},{"label":"Subject","value":["278"]},{"label":"Language","value":["149"]},{"label":"Alternative title","value":["Alternative Title"]},{"label":"Barcode","value":"00000000000000"},{"label":"Folder number","value":"1"},{"label":"Page count","value":["50"]},{"label":"Geographic origin","value":["Spain"]},{"label":"Genre","value":["Banners"]},{"label":"Geo subject","value":["Spain"]},{"label":"Collection","value":["Box Number 1","Latin American Ephemera"]}],"thumbnail":{"@id":"https://libimagestest.princeton.edu/loris/plum/st%2F72%2F2h%2Fc0%2F3g-intermediate_file.jp2/full/!200,150/0/default.jpg","service":{"@context":"http://iiif.io/api/image/2/context.json","@id":"https://libimagestest.princeton.edu/loris/plum/st%2F72%2F2h%2Fc0%2F3g-intermediate_file.jp2","profile":"http://iiif.io/api/image/2/level2.json"}},"seeAlso":{"@id":"https://hydra-dev.princeton.edu/concern/ephemera_folders/sx920fz824.jsonld","format":"application/ld+json"}}'
-    )
+    VCR.turn_off!
+  end
+  after do
+    VCR.turn_on!
   end
   it "converts the JSON-LD into a suitable solr document" do
+    stub_plum_jsonld(record: "b9e8325e-baf2-45e4-b32c-5e5b3755c8ef")
```

Comment:
I've created a ticket to stop using VCR instead of doing some VCR and some webmock stubbing: https://github.com/pulibrary/lae-blacklight/issues/168